README.md

```diff
@@ -8,7 +8,7 @@ This is the CMS for the University of Alberta Libraries website. It uses Comfy M
 
 ## Development environment
 
-+ Ruby 3.1.4
++ Ruby 3.1.4+
```

Comment:
I know we used this convention before in this file, but should we change it to be more specific? Does 3.1.4+ mean that 3.1.4 is included in our acceptable versions or not? Should we change it to `Ruby > 3.1.4` ? Or maybe to how it is specified in the Gemfile `Ruby < 3.5` ?

Maybe this is already a practice and I am missing the mark. What do you think ?

Reply:
I like how it's specified in the Gemfile `Ruby < 3.5`.  I'll make the change.

My rationale:
Pro: 
* shows current version
* can be part of search and replace during updates
* might be more acurate.  We aren't checking if changes are backward compatible with end of life Ruby versions
Cons:
* doesn't show where we came from
* slightly more maintenance/churn because we would need to update each time the Ruby version changes vs this could be forever